lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
+  attr_reader :food_list
+
+  def initialize
+    @food_list = []
```

Comment:
you've made it obvious here that the food_list object is an array - great for the reader!